packages/custom-objects-importer/src/main.js

```diff
@@ -0,0 +1,295 @@
+        if (update) {
+          this._summary.updated += 1
+        } else {
+          this._summary.created += 1
+        }
+        return Promise.resolve()
+      })
+      .catch((error: Error): ExecutionResult => {
+        this._summary.errors.push(error.message || error)
+        let msg
+        if (this.continueOnProblems) {
+          if (update) {
+            this._summary.updateErrorCount += 1
+            msg = 'Update error occurred but ignored. See summary for details'
+          } else {
+            this._summary.createErrorCount += 1
+            msg = 'Create error occurred but ignored. See summary for details'
+          }
+          this.logger.error(msg)
+          return Promise.resolve()
+        }
+        if (update) {
+          this._summary.updateErrorCount += 1
+          msg =
+            'Process stopped due to error while updating custom object. See summary for details'
+        } else {
+          this._summary.createErrorCount += 1
+          msg =
+            'Process stopped due to error while creating custom object. See summary for details'
+        }
+        this.logger.error(msg)
+
+        return Promise.reject(
+          new CustomObjectImportError(
+            msg,
+            this._summary,
+            error.message || error
+          )
+        )
+      })
+  }
+
+  _createOrUpdateObjects(
+    existingObjects: Array<CustomObject>,
+    newObjects: Array<CustomObjectDraft>
+  ): Array<ClientRequest> {
+    this.logger.info('Executing...')
+
+    const createOrUpdateObjects = newObjects.map(
+      (newObject: CustomObjectDraft): CustomObjectDraft | null => {
+        const existing = existingObjects.find(
+          (oldObject: CustomObject): boolean =>
+            oldObject.key === newObject.key &&
+            oldObject.container === newObject.container
+        )
+        if (existing) {
+          if (isEqual(newObject.value, existing.value)) {
+            this._summary.unchanged += 1
+            return null
+          }
+          return { ...newObject, update: true }
+        }
+
+        return newObject
+      }
+    )
+
+    return this._buildRequests(compact(createOrUpdateObjects))
+  }
+
+  _buildRequests(newObjects: Array<CustomObjectDraft>): Array<ClientRequest> {
+    this.logger.info('Creating requests...')
+    const uri = CustomObjectsImporter.buildUri(this.apiConfig.projectKey)
+
+    return newObjects.map((newObject: CustomObjectDraft): ClientRequest =>
+      CustomObjectsImporter.buildRequest(uri, 'POST', newObject)
+    )
+  }
+
+  static buildUri(projectKey: string): string {
+    const service = createRequestBuilder({
+      projectKey,
+    }).customObjects
+    return service.build()
+  }
+
+  static buildRequest(
+    uri: string,
+    method: MethodType,
+    body?: string | Object
+  ): ClientRequest {
+    return body ? { uri, method, body } : { uri, method }
+  }
+
+  _initiateSummary(): Summary {
+    this._summary = {
+      created: 0,
```

Comment:
Yeah, I thought so... it would have been preferable if this was `*Count`

- `createdCount`
- `updatedCount`

Reply:
[fixed!](https://github.com/commercetools/nodejs/pull/651/commits/4ef42ff9ae4cd49886b0c9e979c8872ed21ee8af#diff-65284eab390612879ea62171c18094a3R254)